layer/decorators/layer_wrapper.py

```diff
@@ -12,3 +12,12 @@ def __init__(self, wrapped: Any, wrapper: Any, enabled: Any) -> None:
     # we give it a shot and it seems to be working
     def __reduce_ex__(self, protocol: Any) -> Any:
         return type(self), (self.__wrapped__, self._self_wrapper, self._self_enabled)
+
+    def __copy__(self) -> None:
+        pass
+
+    def __deepcopy__(self, memo: Dict[int, object]) -> None:
+        pass
+
+    def __reduce__(self) -> Union[str, Tuple[Any, ...]]:
+        pass
```

Comment:
Why do we have to put this?

Reply:
Pylint was complaining that these are not implemented but marked as `abstractmethod` in the parent class.